desktop/ui/src/main/java/org/datacleaner/guice/DCModuleImpl.java

```diff
@@ -32,14 +32,7 @@
 import org.apache.metamodel.util.Ref;
 import org.datacleaner.bootstrap.DCWindowContext;
 import org.datacleaner.bootstrap.WindowContext;
-import org.datacleaner.configuration.DataCleanerConfiguration;
-import org.datacleaner.configuration.DataCleanerConfigurationImpl;
-import org.datacleaner.configuration.DataCleanerEnvironment;
-import org.datacleaner.configuration.DataCleanerEnvironmentImpl;
-import org.datacleaner.configuration.DataCleanerHomeFolder;
-import org.datacleaner.configuration.DatastoreXmlExternalizer;
-import org.datacleaner.configuration.InjectionManager;
-import org.datacleaner.configuration.InjectionManagerFactory;
+import org.datacleaner.configuration.*;
```

Comment:
Please import IntelliJ IDEA formatter from the root directory. This change does not comply :)